api/src/main/scala/hmda/api/http/institutions/UploadPaths.scala

```diff
@@ -85,17 +85,17 @@ trait UploadPaths extends InstitutionProtocol with ApiErrorProtocol with Submiss
           case Success(response) =>
             processingActor ! CompleteUpload
             processingActor ! Shutdown
-            complete(ToResponseMarshallable(StatusCodes.Accepted -> Submission(id, Uploaded)))
+            complete(ToResponseMarshallable(StatusCodes.Accepted -> Submission(id, Uploaded, System.currentTimeMillis(), 0L)))
```

Comment:
Now that a start time stamp is being saved for each submission when it is created we should probably return that time here instead of the current time.